README.md

```diff
@@ -54,7 +54,12 @@ We thank the following people for their extensive assistance in the development
 
 --- 
 
-## Data Attribution
+## Licenses and Attribution
+
+This project uses a dual-license setup:
```

Comment:
From what I understood, a "multi-license" setup refers to cases where, for example, software is licensed under different licenses. Not where different components are licensed each with its own license